packages/mds-transaction-api/schema-gen/TransactionApiVersionSchema.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you add a `README` to the `schema-gen` folder to indicate that all of the contained files are automatically generated and should not be edited directly or something like that? Or will the tooling allow `.jsonc` or `.json5`?